Solutions/DigitalGuardianDLP/Analytic Rules/DigitalGuardianUnexpectedProtocol.yaml

```diff
@@ -0,0 +1,28 @@
+id: a14f2f95-bbd2-4036-ad59-e3aff132b296
+name: Digital Guardian - Unexpected protocol
+description: |
+  'Detects unexpected protocol usage.'

```

Comment:
Might be a good idea to give more detail/context in description regarding why the use of Terminal service/RDP is unexpected.

Reply:
changed description